package.json

```diff
@@ -3,48 +3,45 @@
   "version": "2.1.0-alpha.0",
   "description": "React data container for Apollo Client",
   "module": "src/index.ts",
+  "browser": "lib/react-apollo.browser.umd.js",
```

Comment:
@jbaxleyiii I don't understand why this change was necessary. Doesn't it mean that eg. Webpack cannot do tree shaking? Besides it's broken altogether :) #1591 and this is from debugger

![image](https://user-images.githubusercontent.com/1096340/37472607-c87ab09c-286c-11e8-9428-1930743bcc4e.png)